lib/GLPI/Agent/HTTP/Client.pm

```diff
@@ -619,46 +619,35 @@ sub _KeyChain_or_KeyStore_Export {
         my $certdir = $tmpdir->dirname;
         $certdir =~ s{\\}{/}g;
         if (-d $certdir) {
+            my @certCommands = (
+                "certutil -Silent -Split -Store CA",
+                "certutil -Silent -Split -Store Root",
+                "certutil -Silent -Split -Enterprise -Store CA",
+                "certutil -Silent -Split -Enterprise -Store Root",
+                "certutil -Silent -Split -GroupPolicy -Store CA",
+                "certutil -Silent -Split -GroupPolicy -Store Root",
+                "certutil -Silent -Split -User -Store CA",
+                "certutil -Silent -Split -User -Store Root"
+            );
             $logger->debug2("Changing to '$certdir' temporary folder");
             chdir $certdir;
 
+            while (my ($i, $command) = each @certCommands) {
+                my $storeDirname = "$i";
+                mkdir $storeDirname;
+                chdir $storeDirname;
+                getAllLines(
+                    command => $command,
+                    logger  => $logger
+                );
+                chdir "..";
+            }
+
```

Comment:
This each syntax is only supported since perl 5.12 as pointed out by this doc: https://perldoc.perl.org/functions/each

This will work perfectly on windows as we publish glpi-agent with perl 5.38 on this platform. But this means if some one tries to run it on another OS with really old perl version, this will just fail with a syntax error and the problem is this module is used on all OS.

So can I suggest you to modify this part with a more legacy supporting code:
```suggestion
            foreach my $command (@certCommands) {
                my ($kind, $store) = $command =~ /-Split( -\w+)? -Store (\w+)$/;
                my $storeDirname = $kind && $kind =~ /^ -(\w+)$/ ? "$1-$store" : $store;
                mkdir $storeDirname;
                chdir $storeDirname;
                getAllLines(
                    command => $command,
                    logger  => $logger
                );
                chdir "..";
            }

```
Of course, you should test it works as expected ;-)